src/components/entity/ha-statistic-picker.ts

```diff
@@ -39,15 +40,15 @@ export class HaStatisticPicker extends LitElement {
     type: Array,
     attribute: "include-statistics-unit-of-measurement",
   })
-  public includeStatisticsUnitOfMeasurement?: string[];
+  public includeStatisticsUnitOfMeasurement?: string | string[];
 
   /**
    * Show only statistics displayed with these units of measurements.
    * @type {Array}
    * @attr include-display-unit-of-measurement
    */
   @property({ type: Array, attribute: "include-display-unit-of-measurement" })
```

Comment:
We say it is `type: Array`, but it can be a string now too... What does it do with an attribute that is a string?

Reply:
That's handled in _getStatistics by using `ensureArray`:

```ts
      if (includeStatisticsUnitOfMeasurement) {
        const includeUnits = ensureArray(includeStatisticsUnitOfMeasurement);
        statisticIds = statisticIds.filter((meta) =>
          includeUnits.includes(meta.statistics_unit_of_measurement)
        );
      }
      if (includeDisplayUnitOfMeasurement) {
        const includeUnits = ensureArray(includeDisplayUnitOfMeasurement);
        statisticIds = statisticIds.filter((meta) =>
          includeUnits.includes(meta.display_unit_of_measurement)
        );
      }
```

Reply:
No, that is when it is set at property (which we normally do), but what happens when you set a string as attribute, Lit will not try to convert that to an array...

Reply:
(it will try to do a JSON parse, which will fail on a string), we could add our own converter